src/os_posix/os_fallocate.c

```diff
@@ -32,10 +32,13 @@ __posix_std_fallocate(
 	WT_SYSCALL_RETRY(fallocate(pfh->fd, 0, (wt_off_t)0, offset), ret);
 	return (ret);
 #else
+	WT_SESSION_IMPL *session;
+
 	WT_UNUSED(file_handle);
-	WT_UNUSED(wt_session);
 	WT_UNUSED(offset);
-	return (ENOTSUP);
+
+	session = (WT_SESSION_IMPL *)wt_session;
+	return (__wt_set_return(session, ENOTSUP));
```

Comment:
Although this builds, I think my preference is to avoid the local variable in all of these simple cases:
`return (__wt_set_return((WT_SESSION_IMPL *)wt_session, ENOTSUP));`

Reply:
Will do!